test/spec/exprestive_spec.coffee

```diff
@@ -22,7 +22,11 @@ describe 'Exprestive configuration', ->
       expect(@RoutesInitializerMock).to.have.been.calledWith '/base/dir/routes'
 
     it 'uses the default controller directory', ->
-      expect(@ControllerInitializerMock).to.have.been.calledWith '/base/dir/controllers'
+      expectedArg = sinon.match
+        appDir: '/base/dir'
+        controllersPattern: 'controllers/*_controller.{coffee,js}'
+        dependencies: {}
+      expect(@ControllerInitializerMock).to.have.been.calledWith expectedArg
```

Comment:
You could inline the variable `expectedArg` while retaining readability and understandability:

```coffeescript
expect(@ControllerInitializerMock).to.have.been.calledWith sinon.match
  appDir: '/base/dir'
  controllersPattern: 'controllers/*_controller.{coffee,js}'
  dependencies: {}
```